wikitongues/wikitongues/config/load_configs.py

```diff
@@ -92,3 +92,21 @@ def load_languages_airtable_datastores(config):
             config_languages_table['max_records']),
         config_languages_table.getboolean('fake'))
     return languages_datastore
+
+
+def read_include_languages(config):
+
+    if len(config.items("include_languages")) > 0:
+        return config['include_languages']
+
+    iso_codes = [
+        iso for iso in list(config._sections['language_codes'].values())
+    ]
```

Comment:
This part shouldn't be necessary, if we are assuming `include_languages` is a comma-separated list of iso codes. We'll just need to use [`split()`](https://www.w3schools.com/python/ref_string_split.asp) somewhere to convert the string value from the config into a list.

Reply:
Sounds good!